app/lib/katello/resources/candlepin/cpuser.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  module Resources
+    module Candlepin
+      class CPUser < CandlepinResource
```

Comment:
Unfortunately, this file will need to be named `c_p_user.rb` if you want to keep this class name.

Reply:
will see how involved changing it is, I'm not crazy about the name

Reply:
I think we should call it `User` unless that will conflict with another `User` class elsewhere

Reply:
I also wanted to suggest changing to `User`, but... definitely fallback on the `c_p_user.rb` file name if it's crazy to change XD